opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -137,10 +137,23 @@ message Metric {
   //   UpDownSumObserver  Sum(aggregation_temporality=cumulative;is_monotonic=false)
   //   ValueObserver      Gauge()
   oneof data {
+    // (int_gauge and double_gauge) and (int_sum and double_sum) are deprecated,
+    // replaced by gauge and sum fields, respectively.
+    // 1. Old senders and receivers that are not aware of this change will
+    // continue using the four original fields.
+    // 2. New senders, which are aware of this change MUST send only `gauge`
+    // and `sum` fields.
+    // 3. New receivers, which are aware of this change MUST convert this into
+    // `gauge` and `sum` by simply converting all points into Sum or Gauge
+    // metrics using ScalarDataPoint values.
     IntGauge int_gauge = 4;
     DoubleGauge double_gauge = 5;
```

Comment:
Please do the same thing as I did for histogram, refactor the Double version to become the defacto and deprecate only the Int version.

So at least on the receiving side we don't have to do anything for the double version and only translate the in version.

Reply:
PTAL